Bottom: 169d48788fd8587e4cb616027239e8ea997b02f8
Top:    7ee43f726f336c5f608f84409c0ff91687eea140
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:14:09 +0000

w32: TMP Build fixes


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..f6a641c 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -12,6 +9,9 @@ tiff
 freetype
 pixman
 libiconv
+"
+
+BUILD="
 gettext
 glib
 atk
